lib/helpers.js

```diff
@@ -842,6 +842,40 @@ function toAvdLocaleArgs (language, country) {
   return result;
 }
 
+/**
+ * Retrieves the full path to the Android preferences root
+ *
+ * @returns {?string} The full path to the folder or `null` if the folder cannot be found
+ */
+async function getAndroidPrefsRoot () {
+  // https://gerrit.pixelexperience.org/plugins/gitiles/development/+/5c11852110eeb03dc5a69111354b383f98d15336/tools/androidprefs/src/com/android/prefs/AndroidLocation.java
+  let location = null;
+  if (system.isWindows()) {
+    let localAppData = process.env.LOCALAPPDATA;
+    if (!localAppData) {
+      localAppData = process.env.USERPROFILE;
+      if (localAppData) {
+        localAppData = path.resolve(localAppData, 'Local Settings', 'Application Data');
+      }
+    }
+    if (localAppData) {
+      location = path.resolve(localAppData, 'Android');
+    }
+  } else {
+    const home = process.env.HOME;
+    if (home) {
+      location = path.resolve(home, '.android');
+    }
+  }
+
+  if (location && await fs.exists(location)) {
+    const stats = await fs.stat(location);
+    return stats.isDirectory() ? location : null;
```

Comment:
how about print the location as debug log when it is not directory?

Reply:
added log